cmd/experimental/kjobctl/pkg/builder/templates/slurm_init_entrypoint_script.sh.tmpl

```diff
@@ -35,6 +35,30 @@ SBATCH_JOB_NAME={{.SbatchJobName}}
 SBATCH_PARTITION={{.SbatchPartition}}
 EOF
 
+  apk --update add curl jq
```

Comment:
I think it's better to expect the image to have curl and jq then to asume it a `apk` base distribution and do the install for each in it container.

Reply:
It's very hard to find image with `bash`, `curl` and `jq` together.

Reply:
Installing the tools at runtime isn't ideal, for sure. Some users may want to preload images and lock-down the environments from the internet access.

OTOH, we couldn't find any image that would contain all the necessary tools in the official k8s.registry. There are some images on dockerhub, but this might have security risks to use images from third-party vendors, which are not too well maintained.

Also, requiring the user to prepare an image with all the necessary tools isn't ideal, and will be a friction point for the feature.

So, I propose to make the whole feature `SLURM_JOB_FIRST_NODE_IP` opt-in, behind `--first-node-ip` option for now, but keep with the installations. We can warn about the consequences of enabling the feature in help for the option. Long term, we could consider publishing our own image for kjobctl with the necessary tooling.

Reply:
> So, I propose to make the whole feature SLURM_JOB_FIRST_NODE_IP opt-in, behind --first-node-ip option for now, but keep with the installations. We can warn about the consequences of enabling the feature in help for the option. Long term, we could consider publishing our own image for kjobctl with the necessary tooling.

Done